src/assets/scss/components/_search.scss

```diff
@@ -9,4 +9,36 @@
       margin: .25rem .5rem .25rem 0;
     }
   }
+
+  @media only screen and (max-width: 767px) {
+    &__topic {
+      position: relative;
+    }
+
+    &__magnifyingGlass {
+      background-image: url('../img/magnifying-glass.png');
+      background-size: cover;
+      height: 1.75rem;
+      width: 1.75rem;
+      position: absolute;
+      bottom: -.25rem;
+      padding-left: 0;
```

Comment:
@JacOng17 notice how weve duplicated the styles but only added padding-left to both `&__magnifyingGlass` 
We can use `@extend` for that. Take a look here https://sass-lang.com/documentation/file.SASS_REFERENCE.html#extend

Reply:
@JacOng17 Just as reminder as we discovered, we can't use @extend within @media but we can use @mixin :)